microsoft-365/security/office-365-security/mfi-queue-alerts-and-queues.md

```diff
@@ -85,7 +85,7 @@ If you click the number of messages on the widget, a **Messages queued** flyout
 - **Last error**
 - **How to fix**: Common issues and solutions are available. If is a **Fix it now** link is available, click it to fix the problem. Otherwise, click on any available links for more information about the error and possible solutions.
```

Comment:
```suggestion
- **How to fix**: Common issues and solutions are available. If a **Fix it now** link is available, click it to fix the problem. Otherwise, click on any available links for more information about the error and possible solutions.
```